great_expectations/data_context/store/store_backend.py

```diff
@@ -263,6 +264,22 @@ def _has_key(self, key):
     def remove_key(self, key) -> None:
         del self._store[key]
 
+    def build_key(self, name: Optional[str], id_: Optional[str]) -> DataContextKey:
```

Comment:
Promoted to base as it is used throughout the hierarchy

Reply:
Nice! Let's use this instead of the version I added to #5640